blosc/blosc2.c

```diff
@@ -2124,7 +2131,7 @@ int blosc_run_decompression_with_context(blosc2_context* context, const void* sr
 }
 
 
-/* The public routine for decompression with context. */
+/* The public secure routine for decompression with context. */
 int blosc2_decompress_ctx(blosc2_context* context, const void* src, size_t srcsize,
                           void* dest, size_t destsize) {
```

Comment:
For consistency, `srcsize` and `destsize` types should be changed to `int32_t` too.